mips3.asm

```diff
@@ -0,0 +1,21 @@
+# Omer Aldafari		
+# Assembly
+# 2/11/15
+# Part 3		
+			
+	.text	
+	li $t0, 1
+	li $t1, 0
+	li $t3, 15
+	add $t1, $t0, $t1
+
```

Comment:
-6 Instructions require that you store your partial sum in memory and load it again during each iteration.